traffic_portal/conf/config.js

```diff
@@ -21,8 +21,8 @@
 module.exports = {
```

Comment:
I don't think you meant to commit this file

Reply:
oops..